donation.html

```diff
@@ -113,27 +115,23 @@ <h1> Our Story </h1>
                   </p>
                 </div>
                 <div class="col-md-6">
-                  <img class="dono-img" src="./img/donation_page/donation-1.jpg" alt="Technica Fam">
```

Comment:
We might have to adjust the resizing for the images on mobile so that the aspect ratio stays the same